docker/data-science/notebooks/ch03/2.controlling-relevance.ipynb

```diff
@@ -112,8 +85,8 @@
     "    }\n",
     "]\n",
     "print(f\"\\nAdding Documents to '{collection}' collection\")\n",
-    "response = requests.post(f\"{SOLR_URL}/{collection}/update?commit=true\", json=docs).json()\n",
-    "print(\"Status: \" \"Success\" if response[\"responseHeader\"][\"status\"] == 0 else \"Failure\" )\n"
+    "response = engine.add_documents(collection, docs)\n",
```

Comment:
I maybe would move `collection` to the engine constructor, then we could just have `engine.add_documents`

Reply:
Are you referring to collection creation? Coupling the Collection creation w/ the engine constructor creates a situation where a collection is destroyed/recreated anytime an engine object is instantiated

Reply:
Also, an `engine` has multiple collections, so passing in the collection is still necessary to know which one to write to.